daemon/provider/lyric/source/metroprovider.py

```diff
@@ -0,0 +1,34 @@
+import json
+from provider.lyric.lyricparser import LyricParser
+from decorator.singleton import singleton
+
+
+@singleton()
+class MetroProvider(LyricParser):
+
+    # metro domain
+    __metro_domain = "http://www.metrolyrics.com/"
+    # metro result url
+    __metro_lyrics_url = "http://www.metrolyrics.com/{track_name}-lyrics-{artist_name}.html"
+    # metro search api
+    __metro_lyrics_search_url = "http://api.metrolyrics.com/v1/multisearch/all/X-API-KEY/196f657a46afb63ce3fd2015b9ed781280337ea7/format/json?find={artist_name}-{track_name}"
```

Comment:
What's the `X-API-KEY/196f657a46afb63ce3fd2015b9ed781280337ea7`? Will it change someday?